.github/workflows/vulnerability-scan.yml

```diff
@@ -0,0 +1,54 @@
+name: Vulnerability Scan
+
+on:
+  schedule:
+    # Run once every day
+    - cron: '0 0 0 * *'
```

Comment:
I am not sure this is right, https://crontab.guru/#0_0_0_*_* says it is invalid. should it  be `0 0 * * *` ?

Reply:
thanks for the catch!